config/webpack.config.js

```diff
@@ -97,7 +97,7 @@ const commonConfig = function() {
           exclude: /node_modules(?!\/(@novnc|unidiff))/,
           loader: 'babel-loader',
           options: {
-            presets: [require.resolve('@theforeman/builder/babel')],
+            presets: [require.resolve('../webpack/babel')],
```

Comment:
In other places we always use `__dirname`. I'd prefer it if we were consistent and didn't rely on the current working directory too much. Perhaps we need a new `const project_root = path.join(__dirname, '..')` that can then be used everywhere.